app/models.py

```diff
@@ -2665,10 +2665,12 @@ def signed_agreements(self):
         ).join(
             SignedAgreement, User
         ).filter(
-            SignedAgreement.application_id == self.id
+            SignedAgreement.supplier_code == self.supplier_code
+        ).order_by(
+            SignedAgreement.agreement_id
```

Comment:
Should we specify the order direction? e.g. asc or desc

Reply:
this defaults to asc.